about/about-kubernetes-ingress.md

```diff
@@ -0,0 +1,129 @@
+HTTPS requests. The requests are then forwarded via separate connections from the load balancer to the chosen service
+backing pods. As a result, network policy applied to the backing pods can restrict access to only allow connections from the load
+balancer, but cannot restrict access to specific original clients.
+
+### Why use Kubernetes Ingress?
+
+Given that Kubernetes [Services]({{site.baseurl}}/about/about-kubernetes-services) already provide a mechanism for load
+balancing access to services from outside of the cluster, why might you want to use Kubernetes Ingress?
+
+The mainline use case is if you have multiple HTTP / HTTPS services that you want to expose through a single external IP
+address, perhaps with each service having a different URL path, or perhaps as multiple different domains. This is lot
+simpler from a client configuration point of view than exposing each service outside of the cluster using Kubernetes
+Services, which would give each service a separate external IP address.
+
+If on the other hand, your application architecture is fronted by a single "front end" microservice then Kubernetes
+Services likely already meet your needs. In this case you might prefer to not add Ingress to the picture, both from a
+simplicity point of view, and potentially also so you can more easily restrict access to specific clients using network
+policy. In effect, your "front end" microservice already plays the role of Kubernetes Ingress, in a way that is not that
+dissimilar to [in-cluster ingress](#in-cluster-ingress-solutions) solutions discussed below.
+
+### Types of Ingress solutions
+
+Broadly speaking there are two types of ingress solutions:
+- In-cluster ingress - where ingress load balancing is performed by pods within the cluster itself.
+- External ingress - where ingress load balancing is implemented outside of the cluster by
+  appliances or cloud provider capabilities.
+
+#### In-cluster ingress solutions
+
+In-cluster ingress solutions use software load balancers running in pods within the cluster itself. There are many
+different ingress controllers to consider that follow this pattern, including for example the NGINX ingress controller.
+
+The advantages of this approach are that you can: 
+- horizontally scale your ingress solution up to the limits of Kubernetes
+- choose the ingress controller that best suits your specific needs, for example, with particular load balancing
+  algorithms, or security options.
+
+To get your ingress traffic to the in-cluster ingress pods, the ingress pods are normally exposed externally as a
+Kubernetes service, so you can use any of the standard ways of accessing the service from outside of the cluster. A
+common approach is use an external network load balancer or service IP advertisement, with `externalTrafficPolicy:local`.
+This minimizes the number of network hops, and the retains client source IP address, which allows network policy to be used
```

Comment:
Minor: Remove "the" from this section " ... and the retains client source IP address ...".